enum-resource/smuggler.yml

```diff
@@ -0,0 +1,36 @@
+---
+commands:
+  check: |
+    /opt/resource/wrapped/s3/check > response.json
+    # If it is the first run, just dispatch a - string to for 'in' to be triggered
+    jq 'if . == [] then [{ "version_id":"-"}] else . end' < response.json
```

Comment:
In CI it seems that we got the `-` after getting a normal version https://deployer.master.ci.cloudpipeline.digital/pipelines/create-bosh-cloudfoundry/resources/healthcheck-deployed

commenting this with @alext, maybe the issue is that the S3 resource returns a empty list if the current version is the same that is passed as a argument?

We should test that and maybe add a different check. Maybe based on `$SMUGGLER_VERSION_ID`? 

Actually, IIRC S3 check fails if the object is not present.